tests/helpers/test_config_validation.py

```diff
@@ -56,7 +58,54 @@ def test_port():
             schema(value)
 
     for value in ('1000', 21, 24574):
-        schema(value)
+        assert schema(value)
+
+
+def test_isfile():
+    """Validate that the value is an existing file."""
+    schema = vol.Schema(cv.isfile)
+
+    for value in ('invalid', None, -1, 0, 80000, '/fhadh/djd/test.txt'):
+        with pytest.raises(vol.MultipleInvalid):
```

Comment:
balloob mentioned that it is a superclass for my bootstrap PR, and recommended I change it there, so I would say lets stick to `vol.Invalid` everywhere